crates/resolver/src/hosts.rs

```diff
@@ -92,6 +99,7 @@ impl Hosts {
     }
 
     /// parse configuration from `src`
+    #[cfg(any(unix, windows))]
```

Comment:
What's platform-specific about this one?

Reply:
Due to the import:
```
#[cfg(any(unix, windows))]
use proto::rr::{RData, Record};
```
RData cannot be found on non unix/windows

Reply:
I don't see any relevant compile guards on `RData`/`Record`?

Reply:
True. I will remove the guards at the top of the file and on this line too.